python/nbextension/src/widget.ts

```diff
@@ -7,7 +7,7 @@ import './chemiscope-bootstrap.less';
 import 'bootstrap/dist/js/bootstrap.min.js';
 
 import { DefaultVisualizer, MapVisualizer, StructureVisualizer } from '../../../src/index';
-import { Dataset } from '../../../src/dataset';
+import { Dataset, Settings } from '../../../src/dataset';
 
 /**
  * The [[ChemiscopeView]] class renders the Chemiscope App as a widget in the
```

Comment:
Is there a way to create a base class and reduce a bit the repetition? Only if it is a quick fix though

Reply:
yes, I though about this as well. It would involve some form of generics though, since the visualizer has a different type every time. I think this should be alright for now, there is not too much repetition.